tests/components/withings/test_init.py

```diff
@@ -36,7 +36,7 @@ def config_schema_assert_fail(withings_config) -> None:
     """Assert a schema config will fail."""
     try:
         config_schema_validate(withings_config)
-        assert False, "This line should not have run."
+        pytest.fail("This line should not have run.")
```

Comment:
We should probably use `pytest.raises` above instead.

Reply:
Hmmz yeah not exactly, pytest.raises if for when it raises, not when it not raises (as in this case). It is an odd one here (I don't like the approach in general, but... 🤷)

Reply:
`pytest.raises` will error if there's no exception raised. Isn't that what we want? We're expecting `config_schema_validate` to raise an exception here.

Reply:
Oh right... 🤦

Reply:
#86937